UrbanMobility/AccessPoint/doc/spec.md

```diff
@@ -0,0 +1,50 @@
+# gtfs:AccessPoint
+
+## Description
+
+See [https://developers.google.com/transit/gtfs/reference/#stoptxt](https://developers.google.com/transit/gtfs/reference/#stoptxt)
+
+It is a GTFS `stop` which `location_type` is equal to `2`.
+
+## Data Model
+
+### Entity id
+
+It shall be `urn:ngsi-ld:gtfs:AccessPoint:<access_point_identifier>`
+
+being `access_point_identifier` a value that can derived from the `stop_id` field. 
+
+### Entity Type
+
+It shall be equal to `gtfs:AccessPoint` 
+
+### Properties
```

Comment:
this looks not adhering to the current data model template

Reply:
as mentioned below, i still think a should not apply, if not, we wouldn't have any content also for all the GSMA related models. model should be self-descriptive to a minimal level, and beyond that provide references to external pointers. but a minimal level of information should be here, or people will get crazy.

agree on aligning terminology, but that's a different discussion than not trying to re-concile with the existing template, also because we would need that for the existing ones.

+ `ngsiAttributeName` : description.
    + Attribute type: xxx
    + Linked Data type:  xxx
    + Fields in the original model: xxx
    + Optional/Required